notebooks/icrp107_dataset/icrp107_dataset.ipynb

```diff
@@ -1287,7 +1460,7 @@
     "for index in range(0, rows_c.size):\n",
     "    i = rows_c[index]\n",
     "    j = cols_c[index]\n",
-    "    if i == j: c[i,i] = 1.0\n",
+    "    if i == j: continue\n",
     "    else:\n",
     "        sigma = 0.0\n",
     "        for k in rows_dict[j]:\n",
```

Comment:
As the if block is now continue, you don't need to write `else:` and the subsequent statements can be un-indented. I think that will make it easier to read.

Reply:
You can also make identical changes to the three similar blocks to this below.